_includes/intro.html

```diff
@@ -16,9 +16,10 @@ <h2 class="section-headline">TC39</h2>
     <div class="g-col-4">
       <h2 class="section-headline">Contribute</h2>
       <p>
-        TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation,
-        testing, implementations, or even language feature ideas.
-        <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+       TC39 welcomes contributions. You can help by giving proposals feedback, improving documentation, writing tests or
+       implementations, and suggesting language feature ideas.
```

Comment:
Maybe still better to keep this as an `or`? Up to you though.
```suggestion
       implementations, or suggesting language feature ideas.
```